### ISSUE TITLE
Importing gene-sample matrix

### ISSUE THREAD

**Issue by david4096** [ga4gh/server#1515]
Enough RNA seq data is out there in the form of a matrix of sample names to feature identifiers. We should make it easy to present these data over the api without having to write out all the metadata.

**Comment by david4096** [comment]
The issue here is that when we receive a file that is simply a matrix of strings to strings with some value in between, we need to be able to set sane and useful defaults. Using one dimension as feature IDs and the other dimension as RNA Quantifications with expression unit unspecified. Each cell is an expression value

```

                     sample1      sample2
ENST123         1.25             4.56
ENST234          93.4            21.22

```

The only problem I see is that normalization is a boolean, which we may not know. We can implement this feature in the reference server by assuming the values are normalized in this case by default and presenting another flag for setting the normalization if it is known. Perhaps we should suggest a schema change that changes this boolean to an enum to include the unknown case? @saupchurch

**Comment by saupchurch** [comment]
good point @david4096 and one that should probably be extended to any other applicable boolean instances.  Yes, as it is possible to have an "unknown" state we should have the ability to record it as NOT_KNOWN.
